modules/appeals_api/app/sidekiq/appeals_api/appeal_submitted_job.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'feature_flipper'
+
+module AppealsApi
+  class AppealSubmittedJob
+    include Sidekiq::Job
+    # These constants are set to match the previous version of this job, which was AppealsApi::AppealReceivedJob
+    STATSD_KEY_PREFIX = 'api.appeals.received'
+    STATSD_CLAIMANT_EMAIL_SENT = "#{STATSD_KEY_PREFIX}.claimant.email.sent".freeze
+
+    # rubocop:disable Metrics/MethodLength
+    # Sends an email to a veteran or claimant stating that their appeal has been submitted
+    # @param [String] appeal_id The id of the appeal record
+    # @param [String] appeal_class_str The classname of the appeal as a string
+    # @param [String] date_submitted_str The date the appeal was submitted in ISO8601 string format
+    def perform(appeal_id, appeal_class_str, date_submitted_str)
+      return unless FeatureFlipper.send_email?
+
+      if appeal_id.blank? || appeal_class_str.blank? || date_submitted_str.blank?
+        argument_list = [appeal_id, appeal_class_str, date_submitted_str]
+        Rails.logger.error("#{self.class.name}: Missing arguments: Received #{argument_list.join(', ')}")
+        return
+      end
+
+      appeal = appeal_class_str.constantize.find(appeal_id)
+
+      unless appeal.form_data.present? && appeal.auth_headers.present?
+        Rails.logger.error("#{self.class.name}: Missing PII for #{appeal_class_str} #{appeal_id}")
+        return
+      end
+
+      appeal_type_name = appeal.class.name.demodulize.snakecase
+      template_name = "#{appeal_type_name}_received#{appeal.non_veteran_claimant? ? '_claimant' : ''}"
+      template_id = Settings.vanotify.services.lighthouse.template_id[template_name]
+
+      if template_id.blank?
+        Rails.logger.error("#{self.class.name}: could not find VANotify template id for '#{template_name}'")
+        return
+      end
+
+      date_submitted = DateTime.iso8601(date_submitted_str).strftime('%B %d, %Y')
+
+      if appeal.non_veteran_claimant?
+        vanotify_service.send_email(
+          {
+            email_address: appeal.claimant.email,
+            personalisation: {
+              date_submitted:,
+              first_name: appeal.claimant.first_name,
+              veterans_name: appeal.veteran.first_name
+            },
+            template_id:
```

Comment:
My previous version of this in #15661 was passing incorrect arguments to VANotify here. I learned from the original [notify API ruby client docs](https://docs.notifications.service.gov.uk/ruby.html) that the `date_submitted` value (which is part of the data used to fill in the template) needs to be inside `personalisation`, not at the top level.

For reference, you can see the email templates (and the variables they require) by visiting https://notifications.va.gov (on the VA Network only) and logging in with a PIV card.